src/components/TasksGrid/TasksGrid.jsx

```diff
@@ -0,0 +1,67 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+import formatDate from '../../data/formatDate';
+import classes from './TasksGrid.module.css';
+
+const TaskRow = ({ number, progressId, task, startDate, deadlineDate }) => {
+  return (
+    <div className={classes.Row}>
+      <div className={classes.Item}>{number}</div>
+      <Link className={classes.Item} to={`/progress/${progressId}`}>
+        {task}
+      </Link>
+      <div className={classes.Item}>{formatDate(startDate)}</div>
+      <div className={classes.Item}>{formatDate(deadlineDate)}</div>
+    </div>
+  );
+};
```

Comment:
It can be reusable component

Reply:
I created  separated components ```GridRow``` and ```GridCell``` what I use in other components, where I need to display info in form of a grid.
```suggestion
      <GridRow
        rowItemsInfo={rowItemsInfo}
        rowClassName={classes.row}
        cellClassName={classes.cell}
        key={`${start}${index.toString()}`}
      />
```